app/views/important_details/edit.html.erb

```diff
@@ -4,12 +4,23 @@
   <div class="vertical-steps text--centered">
     <div class="vertical-steps__step">
       <h3>Fee agent details</h3>
+      <%= current_interview.fee_agent_name %><br>
+      <%= current_interview.fee_agent_email %><br>
+      <%= format_phone_number(current_interview.fee_agent_phone_number) %>
     </div>
     <div class="vertical-steps__step">
       <h3>Applicant details</h3>
+      <%= current_interview.client_name %><br>
+      XXX-XX-<%= current_interview.client_last_four_ssn %>
     </div>
     <div class="vertical-steps__step">
-      <h3>This application is for</h3>
+      <h3>This <%= application_or_renewal %> is for</h3>
+      <%= raw "SNAP<br>" if current_interview.selected_snap %>
```

Comment:
small thing since we're not displaying user input, but perhaps these should be `.html_safe` rather than `raw`?

Reply:
👍